integration/test_multi_queue_fork_deadlock.py

```
@@ -0,0 +1,54 @@
+
+A write to an indexed key inside MULTI/EXEC is deferred to the multi/exec queue
+and only drained on the next FT.SEARCH. If a fork is in flight at that point the
+mutations thread pool is suspended, and draining it from the main thread never
+completes: the only paths that resume the pool run on that same thread.
+"""
+
+from valkey.client import Valkey
+from valkey_search_test_case import ValkeySearchTestCaseBase
+from valkeytestframework.conftest import resource_port_tracker
+from utils import run_in_thread
+from indexes import *
```

Ruff (0.16.1)

[error] `from indexes import *` used; unable to detect undefined names

(F403)